src/tools/idlc/src/libidlc/libidlc__descriptor.c

```diff
@@ -2587,6 +2587,11 @@ static idl_retcode_t get_ctype_memberids(const idl_pstate_t *pstate, struct desc
     }
   }
 
+  /* TO-DO: XCDR1 for mutable types is currently not supported, so
+     don't include the MID table for mutable types */
+  if (idl_is_extensible(ctype->node, IDL_MUTABLE))
```

Comment:
For mutable we never need it, right? That would make the code is correct also when we do support mutable XCDR1 and the comment would be better phrased as "Mutable encoding doesn't need a MID table".

Reply:
You're right, fixed in  45c2257